src/metabase/pulse.clj

```diff
@@ -63,21 +63,39 @@
                               (str "Pulse: " (:name pulse))
                               attachments)))
 
+(defn is-card-empty?
```

Comment:
I would vote to make this and `are-all-cards-empty?` private since they're only used in this namespace (use `defn-` instead of `defn`)